### ISSUE TITLE
Problem about `restoration_scale`

### ISSUE THREAD

**Issue by mangoyuan** [Fanghua-Yu/SUPIR#58]
when set `s_stage1` = [-1, 0.5, 2.0, 4.0,],  it seems that only s_stage1 output the clear , and others are blur.
In paper Fig13, setting s_stage1 from 0.5, 3, 6 all output clear images.
Is there any mis-understanding about the param?
ths

![截图](https://github.com/Fanghua-Yu/SUPIR/assets/17922959/f6523ace-7de9-42f5-8d08-3eecd815023b)

**Comment by mangoyuan** [comment]
Maybe i found what happen.
In sampling.py, line 408
`denoised = denoised - d_center * ((sigma.view(-1, 1, 1, 1) / self.sigma_max) ** self.restore_cfg)`
At the first of sampling, sigma is near equal to self.sigma_max, so d_center may multiple 1.
```python
d_center = (denoised - x_center)
denoised = denoised - d_center * 1  # denoised = denoised - (denoised - x_center) = x_center, is the latent space of lq image.
```

**Comment by Fanghua-Yu** [comment]
Hello. `s_stage1` is proposed to reduce the over-strong generative abilities of SDXL. Compared to the paper version, we have designed the default sampling parameters more carefully, to make SDXL exhibit properly generative abilities in most cases. Thus `s_stage1` is not needed unless hallucination occurs.

To reproduce `s_stage1`  influences , you can set the output resolution to 512 (we set a min value of 1024 now) and use a fix CFG scale of 7.5.

**Comment by mangoyuan** [comment]
Thx, i test in 512 and fix CFG. It seems that SUPIR can not generate new detail samling from latent representation of LQ.
